score/probes/probes.go

```diff
@@ -16,8 +16,13 @@ func Register(allChecks *checks.Checks, services ks.Services) {
 // ReadinessProbes are not required if the pod is not targeted by a Service.
 //
 // containerProbes takes a slice of all defined Services as input.
-func containerProbes(allServices []corev1.Service) func(corev1.PodTemplateSpec) scorecard.TestScore {
-	return func(podTemplate corev1.PodTemplateSpec) (score scorecard.TestScore) {
+func containerProbes(allServices []corev1.Service) func(corev1.PodTemplateSpec, string) scorecard.TestScore {
+	return func(podTemplate corev1.PodTemplateSpec, kind string) (score scorecard.TestScore) {
+		if kind == "CronJob" {
```

Comment:
With the change suggested in  `checks.go`, this could be changed to also check that the object is in the "batch" group, to not cause conflicts with "CronJobs" in other groups, custom or future built in types.

```suggestion
		if typemeta.Kind == "CronJob" && typemeta.GroupVersionKind().Group == "batch")  {
```